src/Config.cpp

```diff
@@ -238,7 +247,13 @@ void readConfig()
     bool     copy_success = true;                            // will be set to false if copying input names to nameBuffer exceeds array dimensions
                                                              // not required anymore when pins instead of names are transferred to the UI
 
-    if (command == 0)                                        // just to be sure, configLength should also be 0
+    uint16_t length    = MFeeprom.get_length();
+    char     temp      = 0;
+    uint16_t adrPin    = 0;
+    uint16_t adrType   = 0;
+    uint16_t adrConfig = 0;
+
```

Comment:
These are all causing unused variable warnings in the build:

```
src/Config.cpp:250:14: warning: unused variable 'length' [-Wunused-variable]
     uint16_t length    = MFeeprom.get_length();
              ^~~~~~
src/Config.cpp:251:14: warning: unused variable 'temp' [-Wunused-variable]
     char     temp      = 0;
              ^~~~
src/Config.cpp:252:14: warning: unused variable 'adrPin' [-Wunused-variable]
     uint16_t adrPin    = 0;
              ^~~~~~
src/Config.cpp:253:14: warning: unused variable 'adrType' [-Wunused-variable]
     uint16_t adrType   = 0;
              ^~~~~~~
src/Config.cpp:254:14: warning: unused variable 'adrConfig' [-Wunused-variable]
     uint16_t adrConfig = 0;
```

Reply:
I have to check it later

Reply:
Can't remember what I did ;)
Changed it.